Blog Page Created

somshivgupta opened the pull request.

manan-malhotra commented:
Please remove the .idea folder, remove the footer as it is not required in mobile, and put the blog page in (app) folder for routing

somshivgupta commented:
Updated Blog Page by adding report flag, changes in font size and format of Author and Date, adding blog.js into (app) and added library react-native-vector-icons.
